core/src/com/unciv/models/ruleset/Nation.kt

```diff
@@ -84,10 +85,8 @@ class Nation : INamed {
     fun getUniqueString(ruleset: Ruleset): String {
         val textList = ArrayList<String>()
 
-        if (unique != null) {
-            textList += unique!!.tr()
-            textList += ""
-        }
+        textList += unique!!.verbose.tr()
+        textList += ""
```

Comment:
this? :thinking:

Reply:
Not too sure, but was there before my changes so thought I'd leave it in in case it has some obscure use I couldn't find...